newdle/client/src/auth.js

```diff
@@ -14,6 +17,8 @@ export function useAuthentication() {
     if (popup.current) {
       popup.current.close();
```

Comment:
Shouldn't you also dispatch `loginWindowClosed` here if `popupId` has been set? Or is it handled by the `setInterval` callback?

Reply:
We're immediately opening a new window here, so there's no need to dispatch that here. Also, it should be very unlikely that `login()` gets called while another window is open (since we now disable the page with the page dimmer while it's open)

Reply:
FWIW, `login()` can ONLY be called explicitly through user interaction, since any other way of calling it would likely trigger the browser's popup blocker.